### PR TITLE
Updating VS Code extension installation section in readme.md

### PR DESCRIPTION
**Pull request by selimgullulu**
-Added Warning emojis instead of regular exclamation marks -Changed the description for the extension installation which is not automatically done with CFS installation

# Pull Request Template

## Description

-This is for updating the instructions in the readme.md. Suggesting the change of VS Code extension installation section.

## Type of change

Please delete options that are not relevant.

- [X] This change requires a documentation update

Please ensure you have read our CONTRIBUTING.md guide before submitting your pull request.

### CODE REVIEW

**Review by mharringADI — COMMENTED** [review]
Thanks for your first PR! A couple of minor comments before we merge.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Tools VS Code extensions depend on:
 
 #### Install
 
-!!! note
+⚠️
```
[the remark this answers is not in the document]
Comment (selimgullulu) [reply README.md:59]: Thank you, Michael. Just wanted to address some minor issues. 
Kind Regards
Selim

[the remark this answers is not in the document]
Comment (mharringADI) [reply README.md:59]: @selimgullulu we appreciate it! If you make the suggested changes we can merge this PR.

[the remark this answers is not in the document]
Comment (selimgullulu) [reply README.md]: Hi Michael, I updated based on your suggestions. Please kindly check.
Regards
Selim

**Comment by ashish-gawali-adi** [comment]
Looks good!

**Comment by ashish-gawali-adi** [comment]
Looks good! Ship it!

**Review by mharringADI — APPROVED** [review]

**Pull request merged**